docs/source/API/core/policies/RangePolicy.rst

```diff
@@ -11,10 +11,10 @@ Usage
 
 .. code-block:: cppkokkos
 
-    Kokkos::RangePolicy<>(begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(begin, end, args...)
-    Kokkos::RangePolicy<>(Space(), begin, end, args...)
-    Kokkos::RangePolicy<ARGS>(Space(), begin, end, args...)
+    Kokkos::RangePolicy<...>(begin, end)
+    Kokkos::RangePolicy<...>(begin, end, chunk_size)
+    Kokkos::RangePolicy<...>(Space(), begin, end)
```

Comment:
```suggestion
    Kokkos::RangePolicy(exec, begin, end)
```

Reply:
I'm going to have both template parameters and CTAD constructors, as I don't want the usage to imply this is not a templated class.